hub/static/js/explore.esm.js

```diff
@@ -341,12 +342,21 @@ const app = createApp({
             if ( features[layer.feature.properties.PCON13CD] ) {
               var props = features[layer.feature.properties.PCON13CD];
               // "show" the feature
+              var text = "<strong>" + props["name"] + "</strong><br>"
+              for (p in props) {
+                if (["name", "type", "color", "opacity", "PCON13CD"].includes(p)) {
+                  continue
+                }
+                text += p + ": " + props[p] + "<br>"
```

Comment:
If `props[p]` is a float, we should format it to a sensible number of decimal places…

<img width="403" alt="Screenshot 2023-07-06 at 17 15 38" src="https://github.com/mysociety/local-intelligence-hub/assets/739624/5ee9210f-88d1-472a-965c-abaff0a6179f">

And we should also show "%" after it, if it’s a percentage.

Reply:
More generally, I wonder whether we could override the styling in this tooltip, to make it feel more like the rest of the site – `font-size: var(--bs-body-font-size)` and `font-family: var(--bs-body-font-family)` on `.leaflet-tooltip` might do it?

Reply:
Finally, I wonder whether a table layout in here might make it easier to scan down the list of numbers? I’m just thinking, when you have a few datasets active, it’s like a game of hide and seek trying to parse the numbers from each line, whereas a table layout would force the numbers into a neat column down the right-hand side.